components/core/src/clp_s/SchemaTree.hpp

```diff
@@ -93,7 +113,26 @@ class SchemaTree {
         return m_nodes[id];
     }
 
-    int32_t get_root_node_id() const { return m_nodes[0].get_id(); }
+    /**
+     * @return the Id of the root of the Object sub-tree.
+     * @return -1 if the Object sub-tree does not exist.
+     */
+    int32_t get_object_subtree_node_id() const { return m_object_subtree_id; }
+
+    /**
+     * Get the field Id of some field in the Internal subtree.
+     * @param field_name
+     *
+     * @return the field Id of some field in the Internal subtree.
+     * @return -1 if the field does not exist.
```

Comment:
```suggestion
     * Get the field Id for a specified field within the Internal subtree.
     * @param field_name
     *
     * @return the field Id if the field exists within the internal sub-tree, -1 otherwise.
```